hipogram/posts/templates/update.html

```diff
@@ -1,14 +1,20 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+
+
 {% block content %}
 
 <div class="container">
-    <div class="card mb-3">
-        <div class="card-body">
-          <h2 class="card-title">{{ post.created_by.username }}</h2>
-          <img class="img-fluid mb-3" style="max-height:400px" src="{{ post.image.url }}"/>
-          <p class="card-text">{{ post.text }}</p>
-        </div>
-        <small class="card-footer text-muted text-right p-2">{{ post.creation_datetime|date:"H:i - d.m.Y" }}</small>
-      </div>
+  <form method="POST" novalidate>
+      {% csrf_token %}
+      <img class="img-fluid mb-3" style="max-height:400px" src="{{ post.image.url }}"/>
+      <p>Text*</p>
```

Comment:
Adding field label and `*` manually is not a good practice. These should be added dynamically.